developer-docs-site/docs/guides/building-your-own-wallet.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Building Your Own Wallet"
+slug: "building-your-own-wallet"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Building Your Own Wallet
+
+There are already a few wallets building on Aptos. To allow for easy wallet interoperability we need to do the same things across wallets:
+1. Mnemonics
+2. Dapp API
+3. Handling key rotation 
```

Comment:
define these...
 
mnemonics -- a set of key phrases that help derive account private keys
dApp API -- entry points into the wallet to support access to identity managed by the wallet
key rotation -- handling both the relationship around mnemonics and the recovery of accounts in different wallets